src/test/resources/ViewHistory.feature

```diff
@@ -0,0 +1,19 @@
+Feature: View History
+  An a user
+  I want to view history
+  So I can ....
+
+  Scenario: User can book appointment with valid value
+    Given the user login with username as "John Doe" and password as "ThisIsNotAPassword"
```

Comment:
e đã có step `Given user signed in as "John Doe" and "ThisIsNotAPassword"` thì nên sử dụng lai nó, không nên sinh thêm step mới làm gì